assets/js/blocks/attribute-filter/edit.js

```diff
@@ -33,6 +34,14 @@ import { IconExternal } from '../../components/icons';
 import ToggleButtonControl from '../../components/toggle-button-control';
 
 const Edit = ( { attributes, setAttributes, debouncedSpeak } ) => {
+	const {
+		className,
+		heading,
+		headingLevel,
+		showCounts,
+		queryType,
```

Comment:
since we're destructuring all the things here, what about including `attributeId` and `isPreview` as well (see next line)?  This would eliminate additional destructuring in some of the callbacks in this component.

Reply:
Absolutely! Done in 88e4a3bc7aacd5c1ace89e1b9cf30474a145e30e.